bigchaindb/tendermint/lib.py

```diff
@@ -35,17 +35,11 @@ class BigchainDB(object):
     Create, read, sign, write transactions to the database
     """
 
-    BLOCK_INVALID = 'invalid'
-    """return if a block is invalid"""
+    TX_VALID = 'valid'
+    """return if a tx is in valid block"""
 
-    BLOCK_VALID = TX_VALID = 'valid'
-    """return if a block is valid, or tx is in valid block"""
-
-    BLOCK_UNDECIDED = TX_UNDECIDED = 'undecided'
-    """return if block is undecided, or tx is in undecided block"""
-
-    TX_IN_BACKLOG = 'backlog'
-    """return if transaction is in backlog"""
+    TX_UNDECIDED = 'undecided'
+    """return if tx is in undecided block"""
```

Comment:
These are not needed anymore.

Reply:
ah, so the only one that stays is the `TX_VALID`?

Reply:
both `TX_IN_BACKLOG` and `TX_VALID` are needed anymore.

Reply:
ok, will remove it.